File: step30_multiple_stacks/backend/lambda.ts
import { APIGatewayProxyEvent, APIGatewayProxyResult, Context } from 'aws-lambda';
const randomName = require('node-random-name');

exports.handler = async (event: APIGatewayProxyEvent, context: Context): Promise<APIGatewayProxyResult> => {

    const name = randomName();

    console.log("Random Name ==>", name);

    return {
        statusCode: 200,
        headers: {
            'Access-Control-Allow-Origin': '*',
        },
        body: JSON.stringify({
            randomName: name,
        }),
    }
}

File: step30_multiple_stacks/Python/backend/lambda.py
import json
import random
import string


def lambda_handler(event, context):
    # print(event)
    # print(context)
    
    letters = string.ascii_lowercase
    value = ''.join(random.choice(letters) for i in range(10)) 

    return {
        'statusCode': 200,
        "headers":  json.dumps({ 'Access-Control-Allow-Origin': '*' }),
         "body": json.dumps(value) 
    }

